pulsar/schemaDef_test.go

```diff
@@ -0,0 +1,57 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
rename file schema_test.go. We should use snake_case to be consistent with the rest of the files. Same for other camelCase files

Reply:
Yes, this test case is just to test whether the logic of the schemaDef is correct. When we implement the schema function on the producer and consumer respectively, we will add separate test cases for each schema type. At this time, the file will be named `schema_test.go`.

Reply:
we should be consistent with name. all the files are snake_case and I suggest we keep it that way for consistency. schema_def_test.go. Same for the other files in the PR.